src/components/cluster/detail/key_pairs.js

```diff
@@ -140,13 +147,17 @@ class ClusterKeyPairs extends React.Component {
     );
   }
 
-  commonNameFormatter(cell, row) {
+  commonNameFormatter = (cell, row) => {
     return (
-      <Copyable copyText={row.common_name}>
-        <small>{row.common_name}</small>
-      </Copyable>
+      <React.Fragment>
+        <Copyable copyText={row.common_name}>
+          <small>{row.common_name}</small>
+        </Copyable>
+        <br />
+        <Button onClick={this.showKeypairModal.bind(this, row)}>Details</Button>
```

Comment:
I think we have to bite the bullet on this one, since we have to pass a param (row) to this function for every row of the table.

If anyone knows a different way, I'd learn so much if you could explain it.